CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## 1.16.0
+
+- Mark as deprecated
```

Comment:
Is the change to the README what is effectively marking this as deprecated?

Reply:
It's actually [this](https://github.com/lightstep/otel-launcher-python/commit/874258b186209ffeeaf1ee90480acb986154fc18) change.